doc/man/hirte.1.md

```diff
@@ -48,6 +48,7 @@ The level used for logging. Supported values are:
 The target where logs are written to. Supported values are:
 
 - `stderr`
+- `stderr-full`
```

Comment:
For the sake of completeness, this needs to be added to the `doc/man/hirte.conf.5.md` and `doc/man/hirte-agent.conf.5.md` as well.